test/OpenTelemetry.Instrumentation.AWSLambda.Tests/AWSLambdaWrapperTests.cs

```diff
@@ -88,7 +88,7 @@ public void TraceSyncWithInputAndNoReturn(bool setCustomParent)
         {
             var parentContext = setCustomParent ? CreateParentContext() : default;
             AWSLambdaWrapper.Trace(tracerProvider, this.sampleHandlers.SampleHandlerSyncInputAndNoReturn, "TestStream", this.sampleLambdaContext, parentContext);
-            var resource = tracerProvider.GetResource();
+            var resource = tracerProvider?.GetResource();
```

Comment:
I think this is wrong. A NRE would be a much easier to debug error here than what happens if we continue. Instead we should probably add `!` after the `.Build()` for the tracerProvider. Same in other tests.

Reply:
Removed `?`